keps/sig-instrumentation/20190605-metrics-stability-migration.md

```diff
@@ -105,26 +105,33 @@ The [metrics overhaul KEP](https://github.com/kubernetes/enhancements/blob/maste
 
 ## Implementation History
 
-- [x] [Migrate kubelet metrics to use standard prometheus collectors](https://github.com/kubernetes/kubernetes/issues/79286)
-- [ ] Create migrated variants of shared client metrics
-- [ ] Create migrated variants of shared leader-election metrics
-- [ ] Create migrated variants of shared workqueue metrics
-- [ ] Migrate kubelet's /metrics/probes endpoint
-- [ ] Migrate apiserver /metrics endpoint
-- [ ] Migrate scheduler /metrics endpoint
-- [ ] Migrate kube-proxy /metrics endpoint
-- [ ] Migrate controller-manager /metrics endpoint (this include in-tree cloud-provider metrics)
-
-TBD (since this is not yet implemented)
+- [x] [Migrate kubelet metrics to use standard prometheus
+  collectors](https://github.com/kubernetes/kubernetes/issues/79286)
+- [x] Create migrated variants of shared client metrics
+- [x] Create migrated variants of shared leader-election metrics
+- [x] Create migrated variants of shared workqueue metrics
```

Comment:
@logicalhan I couldn't find PRs to link to to check these off, do you have any references?

Reply:
https://github.com/kubernetes/kubernetes/pull/81173/